katas/content/teleportation/entanglement_swapping/Placeholder.qs

```diff
@@ -0,0 +1,21 @@
+namespace Kata {
+    operation EntanglementSwapping() : ((Qubit, Qubit) => (Bool, Bool), (Qubit, (Bool, Bool)) => Unit) {
+        return (SendMessage, ReconstructMessage);
```

Comment:
I think if we return the encoding as integer, we'll be able to make this task a bit more interesting, since it will be more than "just reuse these two operations", it will need to convert the Booleans to an integer and vice versa (and also figure out that that's what you need to do). We can then skip the lambdas, define two operations, SendMessageCharlie and ReconstructMessageBob, with the right signatures and placeholder returns, and have the learner write code in them